server/graphql/invoice/src/mutations/supplier_return/insert.rs

```diff
@@ -100,7 +100,9 @@ fn map_error(error: ServiceError) -> Result<InsertErrorInterface> {
         | ServiceError::OriginalInvoiceNotAnInboundShipment
         | ServiceError::CannotReturnInboundShipment
         | ServiceError::InvoiceAlreadyExists
-        | ServiceError::OtherPartyDoesNotExist => BadUserInput(formatted_error),
+        | ServiceError::OtherPartyDoesNotExist
+        | ServiceError::ManuallyCreatedReturnMustHaveNewStatus
```

Comment:
This should be removed if not used in validation...

Reply:
Thank you for catching this, I missed this.